.github/workflows/validation.yml

```diff
@@ -22,23 +24,22 @@ jobs:
     name: Install framework with restG4
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/lobis/root-geant4-garfield:rest-for-physics
-
+      image: ghcr.io/lobis/root-geant4-garfield:rest-for-physics-dev
     steps:
       - name: Print version of dependencies for image
         run: version.sh
-
       - name: Checkout framework
```

Comment:
For the build and installation of the framework, please consider to use [.github/actions/build/action.yml](https://github.com/rest-for-physics/framework/blob/f881d332a8189967718dcc522aee747d0b7dca07/.github/actions/build/action.yml) from the framework branch.
This is not required in this PR, but we should avoid code duplication also in the workflows.

Reply:
Thanks, I will. Since in this case we only need to checkout a single submodule I think this way is more transparent.